docs/docsite/rst/plugins/inventory.rst

```diff
@@ -80,12 +86,24 @@ You can create dynamic groups using host variables with the constructed ``keyed_
       - key: tags.Name
         prefix: tag_Name_
         separator: ""
+      # If you have a tag called "Role" which has the value "Webserver", this will add the group
```

Comment:
This is demonstrating the same thing as the tags.Name example, so maybe we can improve the wording there.

Reply:
So, `tags.Name` provides an indicator of the role that the "separator" flag provides. Perhaps, instead, we should put something different in there - perhaps using some other tag like this:

```yml
- key: tags.Features
  prefix: Feature
  separator: "%"
```

And show the groups:

```
|--@Feature%Blue
|  |--abc123
|--@Feature%Green
|  |--def456
```

Reply:
OK, so how about this, we yank all of the separator stuff out of here, so it just becomes:

```
- key: tags.Role
  prefix: role
```

(which, I think is generally a better and more accurate example than "this host was named something" as you don't tend to have lots of machines with the same name...)

Then, we fix up the constructed document with some of the examples from here (removing them from this as a result), perhaps including:

```
compose:
  ansible_host: public_ip_address is defined | ternary(public_ip_address, private_ip_address)
  some_inventory_wide_string: '"Yes, you need both types of quotes here"'
```

Reply:
@s-hertel Could you please help here? Thanks in advance.